pages/task/read/index.js

```diff
@@ -17,8 +20,21 @@ module.exports = settings => {
     next();
   });
 
+  app.use('/', (req, res, next) => {
+    if (req.task.data.data.profileId && req.task.data.data.establishmentId) {
+      return req.api(`/establishment/${req.task.data.data.establishmentId}/profile/${req.task.data.data.profileId}`)
+        .then(({ json: { data } }) => {
+          req.profile = cleanModel(data);
+        })
+        .then(() => next())
+        .catch(next);
+    }
+    next();
+  });
+
   app.use('/', form(Object.assign({
     configure: (req, res, next) => {
+      res.locals.static.content = merge({}, getContent(req.task), res.locals.static.content);
```

Comment:
You _probably_ want to swap the order of the last two arguments. The most specific content should be last to be merged on.